recipe/meta.yaml

```diff
@@ -1,16 +1,14 @@
 {% set name = "coveralls" %}
-{% set version = "1.1" %}
-{% set hash_type = "sha256" %}
-{% set hash = "34160385c13b0c43691ab11c080d4b10dabe3280fc0b2173c731efc5db836808" %}
+{% set version = "1.2.0" %}
+{% set git_url = "https://github.com/coveralls-clients/coveralls-python" %}
```

Comment:
For readability I'd paste the URL in the metadata directly without this variable. (And if you follow my suggestion to use the tarball that variable would be used only once, so no need for a variable :smile:)